packages/mds-geography-author/api.ts

```diff
@@ -210,11 +230,15 @@ function api(app: express.Express): express.Express {
   app.put(
     pathsFor('/geographies/:geography_id/meta'),
     checkGeographyAuthorApiAccess(scopes => scopes.includes('geographies:write')),
-    async (req: GeographyAuthorApiRequest, res: PutGeographyMetadataResponse, next: express.NextFunction) => {
+    async (
+      req: GeographyAuthorApiRequest<GeographyMetadata>,
+      res: PutGeographyMetadataResponse,
+      next: express.NextFunction
+    ) => {
       const geography_metadata = req.body
       try {
         await db.updateGeographyMetadata(geography_metadata)
-        return res.status(200).send(geography_metadata)
+        return res.status(200).send({ version: res.locals.version, data: { geography_metadata } })
```

Comment:
I think this was a bug exposed by these type changes. The response payload shape error was not being detected previously.

Reply:
thanks, good catch